redmine/src/test/java/org/arquillian/extension/governor/redmine/impl/reporter/RedmineGovernorRecorderTest.java

```diff
@@ -0,0 +1,140 @@
+
+        KeyValueEntry redmineURL = new KeyValueEntry("Redmine URL","http://redmine.test/issues/2");
+
+        LinkedHashMap<String, String> tableOptions = new LinkedHashMap<String, String>();
+        tableOptions.put("Force","false");
+
+        assertThat(propertyReportEvents).hasSize(2).extracting("propertyEntry.class").contains(KeyValueEntry.class, TableEntry.class);
+        assertThat(propertyReportEvents).extracting("propertyEntry").filteredOn("class", KeyValueEntry.class).contains(redmineURL);
+        assertThat(propertyReportEvents).extracting("propertyEntry").filteredOn("class", TableEntry.class) .contains(getTableEntry("RedmineOptions", tableOptions));
+    }
+
+    @Test
+    public void sholdReportRedmineGovernorParamsForClass(){
+        After after = new After(new FakeTestClass(), getMethod("someTestMethod"));
+        redmineGovernorRecorder.redmineReportEntries(after);
+
+        verify(propertyReportEvent, times(2)).fire(propertyReportEventArgumentCaptor.capture());
+
+        List <PropertyReportEvent> propertyReportEvents = propertyReportEventArgumentCaptor.getAllValues();
+
+        KeyValueEntry redmineURL = new KeyValueEntry("Redmine URL","http://redmine.test/issues/1");
+
+        LinkedHashMap<String, String> tableOptions = new LinkedHashMap<String, String>();
+        tableOptions.put("Force","true");
+
+        assertThat(propertyReportEvents).hasSize(2).extracting("propertyEntry.class").contains(KeyValueEntry.class, TableEntry.class);
+        assertThat(propertyReportEvents).extracting("propertyEntry").filteredOn("class", KeyValueEntry.class).contains(redmineURL);
+        assertThat(propertyReportEvents).extracting("propertyEntry").filteredOn("class", TableEntry.class) .contains(getTableEntry("RedmineOptions", tableOptions));
+
+    }
+
+    private TableEntry getTableEntry(String tableName, LinkedHashMap<String, String> tableOptions){
+        final TableEntry detector = new TableEntry();
+        detector.setTableName(tableName);
+        for (String tableHead: tableOptions.keySet()) {
+            detector.getTableHead().getRow().addCell(new TableCellEntry(tableHead));
+        }
+        TableRowEntry row = contructRow(tableOptions.values());
+        detector.getTableBody().addRow(row);
+
+        return detector;
+    }
+
+    private TableRowEntry contructRow(Collection<String> cellContents){
+        final TableRowEntry row = new TableRowEntry();
+        for (String cellContent: cellContents) {
+            row.addCell(new TableCellEntry(cellContent));
+        }
+        return row;
+    }
+
+    private Method getMethod(String methodName) {
+        Method method = null;
+        try {
+            method = FakeTestClass.class.getMethod(methodName);
+        } catch (NoSuchMethodException e) {
+            e.printStackTrace();
+        }
+        return method;
+    }
+
+    @Redmine(value = "1",  force = true)
+    private static final class FakeTestClass {
+        @Test
```

Comment:
Probably not, but I think this test class is sued as helper inside the enclosing test